### PR TITLE
Add ’make install‘ target

### PR DESCRIPTION
**Pull request by taotieren**
- Add AUR [sunxi-fw-git](https://aur.archlinux.org/packages/sunxi-fw-git)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,3 +15,11 @@ sunxi-%.o: sunxi-%.c
 
 clean:
 	rm -f *.o sunxi-fw
+
+install: sunxi-fw
+	install -Dm755 sunxi-fw $(PREFIX)/bin/sunxi-fw
```
Comment (apritzel) [inline Makefile]: Can you please split the "D" and "m755" up (-D -m755), it looks a bit weird to combine switches and options with arguments. Also please add "-s", to strip the binary before installation.

**Review by apritzel — CHANGES_REQUESTED** [review]
Hi,
thanks for your contribution! I didn't feel like we need an "install" target for a single binary in this stage of the project, but since you want to package it, I guess it doesn't hurt.
Some small change request, and can you please change the commit message to read "add 'make install' target'?
Thanks,
Andre

File: Makefile (hunk 1 above)
Comment (taotieren) [reply Makefile]: fixed

**Comment by taotieren** [comment]
> Hi, thanks for your contribution! I didn't feel like we need an "install" target for a single binary in this stage of the project, but since you want to package it, I guess it doesn't hurt.感谢您的贡献！在项目的这个阶段，我不觉得我们需要单个二进制文件的“安装”目标，但既然你想打包它，我想这不会有什么坏处。 Some small change request, and can you please change the commit message to read "add 'make install' target'?一些小的更改请求，您能否将提交消息更改为“添加'make install'target”？ Thanks, Andre

Thanks for the reply, it has been fixed.

**Pull request merged**